Added Docker Build Functionality

mhanberry1 opened the pull request:
This can now be build using a docker container.

Commit:
Added docker build functionality

Commit:
converted the README file to Markdown

fangohr commented:
Very nice, thank you. I like the comments in the new scripts, thank you.

Adding the rest2www tar file is a good idea I think. 

The tarball and other binaries are currently hosted on mag.soton.ac.uk and we need to migrate them to a new place (which?).

I have added one comment to add that this repo does not contain the documentation, but just the high level webpage overview. (Documentaino is in http://github.com/fangohr/nmag-doc, and [somehow] rendered to http://nmag.readthedocs.io.

Can you update the documentation links to point to http://nmag.readthedocs.io?

Regarding making this compile in a container, this looks good to me.

Commit:
Mention that the docs are built from a separate repository

Commit:
Made the link for the html manual point to http://nmag.readthedocs.io

mhanberry1 commented:
I've changed the links so that they point to http://nmag.readthedocs.io, and I've removed the links for the manual in "multiple html files" because they no longer makes sense.

fangohr commented:
Looks good to me, thanks.
